Apps/Common/src/AccessManagement/Authentication/AuthenticationDelegate.cs

```diff
@@ -126,12 +183,14 @@ private async Task<IAuthModel> ResourceOwnerPasswordGrant(Uri tokenUri, ClientCr
                 content.Headers.Clear();
                 content.Headers.Add(@"Content-Type", @"application/x-www-form-urlencoded");
 
-                using HttpResponseMessage response = await client.PostAsync(tokenUri, content).ConfigureAwait(true);
-
-                string jwtTokenResponse = await response.Content.ReadAsStringAsync().ConfigureAwait(true);
-                this.logger.LogTrace($"JWT Token response: {jwtTokenResponse}");
-                response.EnsureSuccessStatusCode();
-                authModel = JsonSerializer.Deserialize<JWTModel>(jwtTokenResponse)!;
+                using (Task<HttpResponseMessage> task = client.PostAsync(tokenUri, content))
```

Comment:
Nit: the new `using` syntax introduced in C# 8.0 is preferred.  There is also an indentation issue with the block currently.

Reply:
Oddly enough I was getting a disposed error so I switched the style and it worked.  Fixed formatting